### ISSUE TITLE
Revise Descriptions When OSCAL Content Sent by Client

### ISSUE THREAD

**Issue by brian-ruf-ezd** [EasyDynamics/oscal-rest#91]
### Description

As a developer issuing OSCAL REST API calls to send OSCAL content, I need to understand how to identify the OSCAL format I sending.

### Acceptance Criteria

- [ ]  For each model, the description for the following endpoints is updated as defined below
  - [x] `POST /[model-name]`
  - [x] `PUT /[model-name]/{Identifier}`

### Description Content

The current description should be maintained for each of the two endpoints, with the following _added_ to that content:

```
The client must include the `Content-type` HTML header and set it to one of the following:
- `application/json` when sending an OSCAL file in JSON format;
- `application/xml` when sending an OSCAL file in XML format; or
- `application/yaml` when sending an OSCAL file in YAML format.

The implementation _must_ accept all three OSCAL formats. OSCAL content sent to the server in any one of the three formats _must_ be made available in all three formats for the relevant GET method/endpoint combinations.

```

**Comment by brian-ruf-ezd** [comment]
@mpemy This has been addressed for the `POST` methods in PR #72 as required; however, this has not yet been addressed for `PUT /[model-name]/{Identifier}`  
Moving to sprint 67.

**Comment by brian-ruf-ezd** [comment]
`POST` and `PUT` have now both been addressed in PR #72.
